app/service/send_notification.py

```diff
@@ -109,7 +110,7 @@ def send_one_off_notification(service_id, post_data):
         send_notification_to_queue(
             notification=notification,
             research_mode=service.research_mode,
-            queue=template.queue_to_use(),
+            queue=QueueNames.NORMAL if template.queue_to_use() == QueueNames.BULK else template.queue_to_use(),
```

Comment:
Can we please leave a comment on why we want this?

Reply:
Done, please have a look!